client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/site-preview/hooks/use-site-preview-mshot-image-handler.ts

```diff
@@ -89,7 +89,7 @@ export const useSitePreviewMShotImageHandler = ( url: string = '' ) => {
 	}, [ previewRef ] );
 
 	const createScreenshots = ( url: string ) => {
-		const isEnabled = config.isEnabled( 'migration-flow/new-migration-instructions-step' );
+		const isEnabled = getLocaleSlug()?.startsWith( 'en' ); // Previous 'migration-flow/new-migration-instructions-step'.
```

Comment:
Just a small thought, should we use the `import { useIsEnglishLocale } from '@automattic/i18n-utils';` hook instead?

Reply:
I thought about this, and I think it's really the best practice, in case the value changes somehow. But since it shouldn't cause any issue in the short-term, I thought to keep it like it is, just to simplify the code to remove later (in ~1 week). Make sense? If you don't agree, let me know that I can update it. 😉

Reply:
I agree with Imran that we should aim to use the existing hooks as much as possible to keep things DRY, but because this is a short-term thing, it's fine by me. 🙂